src/sdk.js

```diff
@@ -207,9 +207,12 @@ class Smartcar {
    * consented to the exact scope of permission
    * @param {String} [options.vehicleInfo.make] - `vehicleInfo` is an
    * object with an optional property `make`, which allows users to bypass the
-   * car brand selection screen. For a complete list of supported makes, please
+   * car brand selection screen. For a complete list of supported brands, please
    * see our [API Reference](https://smartcar.com/docs/api#authorization)
    * documentation.
+   * @param {Boolean} [options.singleSelect=false] - only allows users to select
```

Comment:
We should not default to false right? like if the developer does not include it, we do not generate the URL with single select. (Api will consider it as false).

Also, the way the code is written now, it would never default to false if you do not include that param .

Reply:
You're right, the JSDoc was incorrect. Fixed!